File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/vssdksupportforsettingscategories/vb/guids.vb
Imports System

Class GuidList
    Private Sub New()
    End Sub

    Public Const guidVSSDKSupportForSettingsCategoriesPkgString As String = "9f31b42d-2206-45c8-bb31-a16117dca29f"
    Public Const guidVSSDKSupportForSettingsCategoriesCmdSetString As String = "565d3359-29b9-4319-86c9-99574711e47d"

    Public Shared ReadOnly guidVSSDKSupportForSettingsCategoriesCmdSet As New Guid(guidVSSDKSupportForSettingsCategoriesCmdSetString)
End Class

File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/vssdksupportforsettingscategories/cs/guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace Microsoft.VSSDKSupportForSettingsCategories
{
    static class GuidList
    {
        public const string guidVSSDKSupportForSettingsCategoriesPkgString = "72c3b4c0-5ef4-4f90-8dea-de3cf6ee032d";
        public const string guidVSSDKSupportForSettingsCategoriesCmdSetString = "91f2346f-08ca-468d-8e4f-484c8f47c43d";

        public static readonly Guid guidVSSDKSupportForSettingsCategoriesCmdSet = new Guid(guidVSSDKSupportForSettingsCategoriesCmdSetString);
    };
}